cmd/kubelet/app/server.go

```diff
@@ -436,8 +436,8 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 	}
 
 	if kubeDeps.ContainerManager == nil {
-		if s.SystemCgroups != "" && s.CgroupRoot == "" {
-			return fmt.Errorf("invalid configuration: system container was specified and cgroup root was not specified")
+		if s.SystemCgroups != "" {
```

Comment:
the Cgroup-Root was checked in function NewContainerManager(),we do not need to do that twice.